python/salpytools/states.py

```diff
@@ -19,8 +19,8 @@
 next_state["ABORT"] = "DISABLED"
```

Comment:
Above here, please add the LSST license header.  The module docstring should use triple double quotes: https://developer.lsst.io/python/numpydoc.html#docstrings-must-be-delimited-by-triple-double-quotes

Reply:
Changed triple ''' to "" and created the following ticket: DM-15863 for LSST license header.